jwql/website/apps/jwql/monitor_pages/monitor_dark_bokeh.py

```diff
@@ -651,8 +651,10 @@ def create_plot(self):
             # for each amp, we can get away with using use_amp=1 at the moment.
             if '5' in self.mean_dark:
                 use_amp = '5'
+                legend_label = 'Full aperture'
             else:
                 use_amp = '1'
+                legend_label = 'Amp 1'
```

Comment:
This still needs to be tested on the test server.

Reply:
Done. Looks good.

Reply:
Thanks!